src/iptc.cpp

```diff
@@ -354,7 +354,10 @@ namespace Exiv2 {
 		while  ( i < size-3 && bytes[i] != 0x1c ) i++;
 		depth++;
 		out << Internal::indent(depth) << "Record | DataSet | Name                     | Length | Data" << std::endl;
-		while ( bytes[i] == 0x1c && i < size-3 ) {
+		while ( i < size-3 ) {
```

Comment:
The indentation now looks weird in this function. The original code was using tabs and now you started to use spaces. Would you mind to convert all these tabs in spaces?

Reply:
I have clang-formatted the whole function in a separate commit.